includes/class-wc-stripe.php

```diff
@@ -857,4 +860,39 @@ public function initialize_status_page() {
 		$wcstripe_status = new WC_Stripe_Status( self::get_main_stripe_gateway(), $this->account );
 		$wcstripe_status->init_hooks();
 	}
+
+	/**
+	 * Maybe deactivate Affirm or Klarna payment methods if other official plugins are active.
+	 *
+	 * @return void
+	 */
+	public function maybe_deactivate_bnpls() {
+		$has_affirm_plugin_active = WC_Stripe_Helper::has_gateway_plugin_active( WC_Stripe_Helper::OFFICIAL_PLUGIN_ID_AFFIRM );
+		$has_klarna_plugin_active = WC_Stripe_Helper::has_gateway_plugin_active( WC_Stripe_Helper::OFFICIAL_PLUGIN_ID_KLARNA );
+		if ( ! $has_affirm_plugin_active && ! $has_klarna_plugin_active ) {
+			return;
+		}
+
+		$gateway = $this->get_main_stripe_gateway();
```

Comment:
Should we verify that we have the UPE gateway before we proceed? Users can technically still use the old gateway.